src/services/activityReports.js

```diff
@@ -328,7 +328,7 @@ export function activityReports(readRegions, {
   };
   return ActivityReport.findAndCountAll(
     {
-      where: { regionId: regions },
+      where: { regionId: regions, status: REPORT_STATUSES.APPROVED },
```

Comment:
I believe we need to show "submitted" reports in the submitter's alerts table now, or they'll fall off any display until the approver looks at them.